xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -335,6 +335,17 @@ bool CGUIWindowPVRGuideBase::OnMessage(CGUIMessage& message)
     {
       if (message.GetSenderId() == m_viewControl.GetCurrentControl())
       {
+        if (message.GetParam1() == ACTION_SELECT_ITEM ||
```

Comment:
Could this be handled in the base class so the code wouldn't have to be duplicated?

Reply:
I suggest to keep it as it is, as only two out of five pvr window classes need this.

Reply:
Sure